### PR TITLE
Support p4runtime info for serenum action arguments

### LINKED ISSUE #2904: Binding action args to serializable enums
It appears that compiler does not support binding of action args to simple enum types. This helps with readability.

Ask is to support accepting serializable enum types as action arguments.

```
enum bit<5> TableType {
  TT_ACL = 0,
  TT_FWD = 1
};

action A(TableType t) {
  <some_metadata> = t;
}

table X {
   key = {
      meta.m: ternary;
  }
  actions = {
    A;
  }
  const entries = {
    (0x0): A(TableType.TT_ACL);
    (0x1): A(TableType.TT_FWD);
    (_): A(TableType.TT_FWD);
  }
}
```

### PR DESCRIPTION
**Pull request by mbudiu-vmw**
Fixes #2904

### CODE REVIEW

File: control-plane/p4RuntimeSerializer.cpp [hunk 1]
```diff
@@ -1539,12 +1539,17 @@ class P4RuntimeEntriesConverter {
...
             if (arg->expression->is<IR::Constant>()) {
                 auto value = stringRepr(arg->expression->to<IR::Constant>(), width);
                 protoParam->set_value(*value);
             } else if (arg->expression->is<IR::BoolLiteral>()) {
                 auto value = stringRepr(arg->expression->to<IR::BoolLiteral>(), width);
                 protoParam->set_value(*value);
+            } else if (ei != nullptr && ei->is<SerEnumInstance>()) {
+                auto sei = ei->to<SerEnumInstance>();
+                auto value = stringRepr(sei->value->to<IR::Constant>(), width);
+                protoParam->set_value(*value);
```
Comment (antoninbas) [inline control-plane/p4RuntimeSerializer.cpp:1552]: I notice that in other places, we only call `EnumInstance::resolve` if the expression is actually an `IR::Member`, but I don't know if one approach is better than the other.

**Review by antoninbas — APPROVED** [review]
LGTM

File: control-plane/p4RuntimeSerializer.cpp (hunk 1 above)
Comment (mbudiu-vmw) [reply control-plane/p4RuntimeSerializer.cpp:1552]: I think this is safe - if it's not member, it won't resolve...
Thanks.

**Review by jafingerhut — COMMENTED** [review]
Already merged, but I also checked the P4Info and entries files generated by the compiler, and they look correct for the input program.  Thanks!

**Pull request merged**